### PR TITLE
[ENH] Refactored series_length and n_instances to n_timepoints and n_cases

### LINKED ISSUE #1153: [ENH] Refactor `series_length` and `n_instances` to `n_timepoints` and `n_cases` in convolution based classifiers
### Describe the feature or idea you want to propose

some time ago we agreed on calling the shape of collections of time series `(n_cases, n_channels, n_timepoints)`. Some classes still use `n_instances` and `series_length`, which is fine, but we would like to standardise.

### Describe your proposed solution

_arsenal.py and _rocket classifier contain `n_instances` and/or `series_length`  in docstrings and as variable names. Refactor for `n_cases` and `n_timepoints`. 

### Describe alternatives you've considered, if relevant

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Raya679**
Fixes #1153 

I have refactored series_length and n_instances to n_timepoints and n_cases in convolution based classifiers
Please let me know if any more changes are required

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/aeon-toolkit/aeon/blob/main/.all-contributorsrc). Alternatively, you can use the [@all-contributors](https://allcontributors.org/docs/en/bot/usage) bot to do this for you.
- [x] The PR title starts with either [ENH], [MNT], [DOC], [BUG], [REF], [DEP] or [GOV] indicating whether the PR topic is related to enhancement, maintenance, documentation, bugs, refactoring, deprecation or governance.

### CODE REVIEW

File: aeon/classification/convolution_based/_arsenal.py [hunk 1]
```diff
@@ -146,9 +146,9 @@ def __init__(
         self.random_state = random_state
         self.n_jobs = n_jobs
 
-        self.n_instances_ = 0
+        self.n_cases_ = 0
         self.n_dims_ = 0
```
Comment (TonyBagnall) [inline aeon/classification/convolution_based/_arsenal.py:150]: as you are here, could you change n_dims to n_channels? thanks

**Review by TonyBagnall — COMMENTED** [review]
looks good, thanks for the PR, but as you are here, could you change n_dims to n_channels?

File: aeon/classification/convolution_based/_arsenal.py (hunk 1 above)
Comment (Raya679) [reply aeon/classification/convolution_based/_arsenal.py]: I have made the required changes as mentioned. Please let me know if any more changes are required. 
Thank you!

**Review by TonyBagnall — APPROVED** [review]
LGTM

**Comment by MatthewMiddlehurst** [comment]
@all-contributors add @Raya679 for docs

**Pull request merged**